gammapy/time/plot.py

```diff
@@ -2,20 +2,18 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-__all__ = ['plot_time_difference_distribution',
-           ]
 
+__all__ = ['plot_light_curve',
+           ]
 
 def plot_time_difference_distribution(time, ax=None):
     """Plot event time difference distribution.
-
```

Comment:
Put the empty line back in.
(I think it's actually required for the docstring to be processed correctly by Sphinx)